samples/SnapshotConsoleSample/SnapshotConsoleSample.c

```diff
@@ -485,15 +580,16 @@ int main(int argc, char *argv[]) {
 			"                        BZX, DEA, ISE, DEX, IST, ...\n"
 			"                     b) source for MarketMaker, one of following: AGGREGATE_BID\n"
 			"                        or AGGREGATE_ASK\n"
-			"  " TIME_PARAM_SHORT_TAG " <DD-MM-YYYY>  - Time from which to receive data (default: current date and\n"
-			"                     time)\n"
+			"  " FROM_TIME_PARAM_SHORT_TAG " <DD-MM-YYYYTHH:mm:ss>  - Time from which to receive data (default: current date and time\n"			
+			"  " TO_TIME_PARAM_SHORT_TAG "   <DD-MM-YYYYTHH:mm:ss>  - Time to which to receive data (default: infinity, exist only for CANDLE, TIME_AND_SALE and GREEKS)\n"
 			"  " RECORDS_PRINT_LIMIT_SHORT_PARAM " <limit>       - The number of displayed records "
 			"(0 - unlimited, default: " STRINGIFY(DEFAULT_RECORDS_PRINT_LIMIT)")\n"
 			"  " TOKEN_PARAM_SHORT_TAG " <token>       - The authorization token\n"
 			"  " LOG_DATA_TRANSFER_TAG "               - Enables the data transfer logging\n"
 			"  " TIMEOUT_TAG " <timeout>     - Sets the program timeout in seconds (default = 604800,\n"
 			"                     i.e a week)\n"
-			"Example: demo.dxfeed.com:7300 ORDER IBM NTV -t 01-01-1970 -o 30\n\n"
+			"Example: demo.dxfeed.com:7300 ORDER IBM NTV -from-time 01-01-1970T00:00:00 -o 30\n\n"
```

Comment:
We should specify that user must don't use `-from-time` `-to-time` with "indexed only" (indexed but not time series) events (Order, Spread Order, Series).

Reply:
Done